src/ui/PipelineContext/PipelineContext.ts

```diff
@@ -44,63 +46,97 @@ export interface IPipelineContextOptions {}
  * Using this component as opposed to JavaScript code means you will be able to leverage the interface editor.
  *
  * Regardless of if you use this component or JavaScript to add context, both will add the needed data in the [Query.Context]{@link IQuery.context} parameter.
+ * 
+ * **Note:**
+ * 
+ * Using this component also ensures that the framework will be able to properly determine the context in all corner cases, including when using a Standalone Search box ({@link initSearchbox}) displaying query suggestions.
+ * 
+ * If this component is not used in those cases, the context won't be able to be resolved and leveraged properly in the [Query Pipeline](http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=108).
```

Comment:
I'd rephrase as follows:

> In most cases, if you do not use this component, the context will not be resolved and leveraged properly in the query pipeline (see [What Is a Query Pipeline?](http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=252)).

Note that I also changed the link to an introductory topic on query pipelines.